src/ASM/ASMs3DmxLag.C

```diff
@@ -596,7 +603,7 @@ bool ASMs3DmxLag::evalSolution (Matrix& sField, const IntegrandBase& integrand,
 
           // Compute Jacobian inverse of the coordinate mapping and
           // basis function derivatives w.r.t. Cartesian coordinates
-          if (!fe.Jacobian(Jac,Xnod,dNxdu,geoBasis))
+          //if (!fe.Jacobian(Jac,Xnod,dNxdu,geoBasis))
```

Comment:
This should not have been commented out (an error or mine, I guess). Should be
`if (!fe.Jacobian(Jac,Xnod,geoBasis,nullptr,&dNxdu))`

Reply:
let's collectively take the blame ;)

what scares me more is no tests failed.

Reply:
probably the 3D mixed Lagrange is not covered by any regression test, at least not when it comes to solution evaluation.